### ISSUE TITLE
form button ambiguous

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/tripal_cv_xray#40]
When at the form, `admin/tripal/extension/tripal_cv_xray`, the button says Populate All, even with nothing selected.
Whats the expected/intended behavior here? Currently ambiguous.

<img width="616" alt="screen shot 2018-06-19 at 1 00 59 pm" src="https://user-images.githubusercontent.com/7063154/41612414-dbbe0010-73c0-11e8-97ca-2dc753e5d257.png">

**Comment by almasaeed2010** [comment]
Should it be changed to "Populate Index"?

**Comment by bradfordcondon** [comment]
yes but also need to make it clear that running it will remove all other previously indexed configs.
